export_test.go

```diff
@@ -0,0 +1,5 @@
+package newton
+
+const (
+	BaseURL = baseUrl
+)
```

Comment:
I don't think this file is needed, we can use `New` in the auth test and for verifying the prod url, I prefer to have another const in the test package, so if someone change the prod url he need to change it in 2 places to make sure it's a willing change.